Scene/scene.js

```diff
@@ -74,6 +78,18 @@ class Scene {
         return id;
     }
 
+    async UpdateResolution(resolution) {
+        let ids = Object.keys(this._models);
+        let models = [];
+        for (const id of ids) {
+            models.push(this._models[id]);
+        }
+        for (const model of models) {
+            await this.AddToScene(model.model.source, model.startTime, model.endTime, model.cadence, GetImageScaleForResolution(resolution, model.model.source), model.order);
```

Comment:
Can you take the result of this and assign it back to the original `id` index?
This needs to be done so that the front end controls can still talk to the scene.

When first you add a source to the scene it will have `id = 1` So things like locking the camera or changing the opacity on it make calls to with `id = 1`.

When the resolution changes and this function is called it removes the old model and adds a new one with the new resolution, now the `id = 2` (or really `id = N` depending on how many models are in the scene), the front end doesn't (and shouldn't) know that anything has changed, so you need to make sure that calls with `id = 1` still act on this same data source.

Should be as simple as
```
let new_id = await this.AddToScene(...)
this._models[original_id] = this._models[new_id]
del this._models[new_id]
```